test/ruby/test_ast.rb

```diff
@@ -1338,6 +1338,14 @@ def test_unless_locations
       assert_locations(node.children[-1].locations, [[1, 0, 1, 10], [1, 2, 1, 8], nil, nil])
     end
 
+    def test_groval_valias_locations
+      node = RubyVM::AbstractSyntaxTree.parse("alias $foo $bar")
+      assert_locations(node.children[-1].locations, [[1, 0, 1, 15], [1, 0, 1, 5]])
+
+      node = RubyVM::AbstractSyntaxTree.parse("alias $foo $& if 'bar' =~ /baz/")
```

Comment:
Is `if 'bar' =~ /baz/` needed?

Reply:
Indeed it is. I guess I was mistaken. I have removed `if 'bar' =~ /baz/`.